cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -217,10 +220,11 @@ func RelayBatchProphecyCompletedToEthereum(
 
 		if err != nil {
 			sugaredLogger.Errorw("Failed to submit to ethereum client", "error", err)
-			sleepThread(1)
 		} else {
 			break
 		}
+		// exponential backoff
+		sleepDuration *= sleepDuration
```

Comment:
I dont think this is the right implementation

```
i | sleepDuration value at end of iteration
0 | 2 * 2 = 4
1 | 4 * 4 = 16
2 | 16 * 16 = 256 
3 | 256 * 256 = 65536 = 1092 minutes
4 | 65536 * 63356 = 49710 days

Reply:
I think you want 
```
sleepDuration *= 2

Reply:
will set maxRetries as 4, last sleep about 4 min, then 
i | sleep | sleepDuration after sleep
0 | 2 | 4
1 | 4 | 16
2 | 16 | 256
3 | 256 | 65536
4 exit from loop